subsys/bluetooth/mesh/Kconfig

```diff
@@ -289,6 +289,19 @@ config BT_MESH_CRPL
 	  protection list. This option is similar to the network message
 	  cache size, but has a different purpose.
 
+choice BT_MESH_RPL_STORAGE_MODE
+	prompt "Replay protection list storage mode"
+	default BT_MESH_RPL_STORAGE_MODE_SETTINGS
+
+config BT_MESH_RPL_STORAGE_MODE_SETTINGS
```

Comment:
Since RPL is not stored by timeout if this option is disable, `BT_MESH_RPL_STORE_TIMEOUT` should either be moved under this option or switched on only if this option is enabled,